File: modules/jooby-openapi/src/test/java/issues/i1905/SubApp1905.java
/*
 * Jooby https://jooby.io
 * Apache License Version 2.0 https://jooby.io/LICENSE.txt
 * Copyright 2014 Edgar Espina
 */
package issues.i1905;

import io.jooby.Jooby;

public class SubApp1905 extends Jooby {
  {
    get("/sub", ctx -> "OK");
  }
}

File: modules/jooby-openapi/src/test/kotlin/kt/i1905/SubApp1905.kt
/*
 * Jooby https://jooby.io
 * Apache License Version 2.0 https://jooby.io/LICENSE.txt
 * Copyright 2014 Edgar Espina
 */
package kt.i1905

import io.jooby.Context
import io.jooby.kt.Kooby

class SubApp1905 : Kooby({ get("/sub") { _: Context? -> "null context" } })
